en-US/Characters_Codex/06_Spellcasting/Spell_Descriptions/J/Jesss_Private_Sanctum.md

```diff
@@ -1,23 +1,43 @@
 #### Jess's Private Sanctum
 <!-- previously "Private Sanctum" -->
-<!-- TODO Check and tag this spell -->
+<!-- markdownlint-disable link-image-reference-definitions -->
+[_metadata_:spell_name]:- "Jess's Private Sanctum"
+[_metadata_:spell_original_name]:- "Private Sanctum"
+[_metadata_:spell_level]:- "4"
+[_metadata_:spell_school]:- "abjuration"
+[_metadata_:ritual]:- "false"
+[_metadata_:casting_time_amount]:- "10"
+[_metadata_:casting_time_unit]:- "minutes"
+[_metadata_:range]:- "120 feet"
+[_metadata_:target]:- "a cube with sides between 5 and 100 feet long"
+[_metadata_:components_verbal]:- "true"
+[_metadata_:components_somatic]:- "true"
+[_metadata_:components_material]:- "true"
+[_metadata_:components_material_description]:- "a thin sheet of lead, a piece of opaque glass, a wad of cotton or cloth, and powdered chrysolite"
+[_metadata_:duration]:- "24 hours"
+[_metadata_:concentration]:- "false"
+[_metadata_:compared_to_wotc_srd_5.1]:- "mechanics_same_wording_different"
+[_metadata_:compared_to_a5e_srd]:- "mechanics_different_wording_different"
 <!-- markdownlint-disable-next-line no-emphasis-as-heading -->
+_4th-level abjuration_
 
 **Casting Time:** 10 minutes \
 **Range:** 120 feet \
 **Components:** V, S, M (a thin sheet of lead, a piece of opaque glass, a wad of cotton or cloth, and powdered chrysolite) \
 **Duration:** 24 hours
 
-You increase the magical security in an area, choosing one or more of the following:
+You increase the magical security in a cube with sides between 5 and 100 feet long, choosing one or more of the following:
 
 - Sound cannot pass the edge of the area.
-- Light and vision cannot pass the edge of the area.
+- Light and vision (including [Darkvision](#Exploration_Environment_darkvision)) cannot pass the edge of the area.
 - Sensors created by divination spells can neither enter the area nor appear within it.
 - Creatures within the area cannot be targeted by divination spells.
 - Nothing can teleport into or out of the area.
 - Planar travel is impossible within the area.
 
+You may dismiss this spell as an action.
```

Comment:
This is part of the WotC version, but not the A5e version. Since it is not a concentration spell and you normally can't end spells with an action, I thought it was important to add that back in.